packages/app-store/zoomvideo/lib/VideoApiAdapter.ts

```diff
@@ -233,11 +269,20 @@ const ZoomVideoApiAdapter = (credential: CredentialPayload): VideoApiAdapter =>
       },
       isAccessTokenUnusable: async function (response) {
         const myLog = logger.getSubLogger({ prefix: ["zoomvideo:isAccessTokenUnusable"] });
-        myLog.debug(safeStringify({ status: response.status, ok: response.ok }));
-        if (!response.ok || (response.status < 200 && response.status >= 300)) {
```

Comment:
response.status check had always been wrong. The condition is never truthy(Nothing is less than 200 and >300 simultaneously)

response.ok false already takes care of status code check and thus we can remove explicit status check